webpack.config.js

```diff
@@ -78,6 +84,8 @@ module.exports = {
     devtool: 'source-map',
 
     performance: {
+        maxAssetSize: 670 * 1024,
+        maxEntrypointSize: 670 * 1024,
```

Comment:
Can you add comments about what these lines do?

Reply:
What would it say? The whole config is documented on https://webpack.js.org/configuration/performance/

The default value is 250kb which we are now exceeding, so we have to explicitly define these.

Reply:
That's fair; maybe add that link to the `performance` object? Build config files are kind of difficult to grok, so making it easier for future folk to understand what's happening makes them less intimidating.

Reply:
I've linked to https://webpack.js.org/configuration. See comment below